image/Dockerfile.laboratory

```diff
@@ -0,0 +1,14 @@
+FROM node:9.11-alpine
+
+WORKDIR /opt
+
+RUN apk add -qU --no-cache -t .devstuff git \
+    &&  git clone https://github.com/stellar/laboratory.git \
+    &&  cd /opt/laboratory \
+    &&  npm install \
+    &&  apk del -q .devstuff
+WORKDIR /opt/laboratory
+COPY /laboratory/configure_network.sh .
```

Comment:
mount `image/laboratory` instead of baking it into the image
```Dockerfile
VOLUME ["/opt/laboratory"]
```
and in docker-compose mount:
```yaml
    volumes:
      - ./laboratory/opt/laboratory:/opt/laboratory
```